compiler/damlc/daml-stdlib-src/Prelude.daml

```diff
@@ -16,3 +16,4 @@ import DA.Internal.Template.Functions as X
 import DA.Internal.Compatible as X
 import DA.Internal.Assert as X
 import DA.Internal.Interface as X
+import DA.Internal.Interface.AnyView.Types as X
```

Comment:
Needed to make `AnyView` accessible to `LowLevel`

Reply:
We cannot import Internal package in arbitrary packages.

Reply:
Moved import to `LowLevel` only

Reply:
Ah, `checkImports` only forbids _unstable_ internal modules (`isUnstableInternal`). Since `DA.Internal.Interface.AnyView.Types` is internal _but_ stable, the preprocessor allows it. I think that makes sense, users can't cause too much trouble by importing internal stable packages, since they just define data structures.

https://github.com/digital-asset/daml/blob/6fa4abf91b6a6a2ff74055098f77ce4a415d4c8c/compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs#L174-L177
https://github.com/digital-asset/daml/blob/6fa4abf91b6a6a2ff74055098f77ce4a415d4c8c/compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs#L53-L56